src/Akka.Streams.Kafka/Messages/CommittableOffset.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Akka.Streams.Kafka.Stages.Consumers;
+
+namespace Akka.Streams.Kafka.Messages
+{
+    /// <summary>
+    /// Implementation of the offset, contained in <see cref="CommittableMessage{K,V}"/>.
+    /// Can be commited via <see cref="Commit"/> method.
+    /// </summary>
+    internal sealed class CommittableOffset : ICommittableOffsetMetadata
+    {
+        /// <inheritdoc />
+        public long BatchSize => 1;
```

Comment:
Batchsize can't be bigger than 1?

Reply:
Ah nevermind, I see why now.

Reply:
Yes, just a special value for this offset implementation